docs/spec.md

```diff
@@ -154,6 +154,7 @@ before their uses. Possible execution orders of the example program above are
 → `return`.
 
 More formally, a **StableHLO process** is a combination of:
+
 1) a StableHLO program, 2) operation statuses (not executed yet,
```

Comment:
Actually this was not supposed to be a list, just a regular paragraph. How do we express that? I guess, "1\)"?

Reply:
Ah. Yes, full parentheses solves it. That is: "(1)"